src/GMBuddy.Games/Micro20/OutputModels/CharacterSheet.cs

```diff
@@ -87,6 +96,58 @@ public int Mind
                 }
             }
         }
+
+        public int Physcial
+        {
+            get
+            {
+                int rank = c.Level;
+                if(c.Class == Micro20ClassType.Fighter)
```

Comment:
No need to go through and change all of them, but in general I really strongly discourage the use of any blocks without curly braces. I feel like it doesnt make anything except inserting bugs easier.